app/services/meeting_starter.rb

```diff
@@ -21,7 +21,13 @@ def call
 
       @room.update!(online: true, last_session: DateTime.strptime(meeting[:createTime].to_s, '%Q'))
 
-      ActionCable.server.broadcast "#{@room.friendly_id}_rooms_channel", 'started'
+      # new thread to avoid blocking the other code from running
+      Thread.new do
+        sleep 7 # Sleep 7 seconds to allow enough time for room to start
+        ActionCable.server.broadcast "#{@room.friendly_id}_rooms_channel", 'started'
+        sleep 7 # Sleep and rebroadcast again incase anyone missed the first broadcast
+        ActionCable.server.broadcast "#{@room.friendly_id}_rooms_channel", 'started'
+      end
```

Comment:
Wait 7 seconds and then inform the waiting cables/sockets that the meeting has started, and the wait another 7 seconds and broadcast again (incase we missed someone)

This is running in a new thread to prevent the whole app from waiting (due to the sleep)